charts/substra-backend/templates/deployment-server.yaml

```
@@ -12,6 +12,8 @@ spec:
     matchLabels:
         app.kubernetes.io/name: {{ template "substra.name" . }}-server
         {{ include "substra.selectorLabels" . | nindent 8}}
+  strategy:
+    type: Recreate
```

Wdyt of adding the field in the values file (just like the registry):
```suggestion
  strategy:
    type: {{ .Values.server.updateStrategy }}
```

---

I can make it configurable through the values if you think it's cleaner, I don't see any case where we wouldn't want to use the `Recreate` strategy since it's blocking the deployment because of the shared RWO PVC.
For the registry, we use an external chart, so we have to configure it through values.
No strong opinion in either direction though.